app/src/main/java/com/example/ncov19traking/ui/global/GlobalFragment.kt

```diff
@@ -114,5 +157,11 @@ class GlobalFragment : Fragment() {
         val deathsPercentage: TextView = view.findViewById(R.id.percentageDeathsDifference)
         val lastUpdateLong: TextView = view.findViewById(R.id.last_update_date)
         val progressBar: ProgressBar = view.findViewById(R.id.progressBar)
+        val pieChart: PieChart = view.findViewById(R.id.circular_chart)
     }
 }
+
+fun formatLargeNumbers(value: Int): String {
+    val df = DecimalFormat("###,###,###")
+    return df.format(value)
+}
```

Comment:
this should not be in the fragment. It is public and it doesn't have anything to do with the fragment, It should be moved to an utility formatter file. This way you could also reuse it somewhere else in your project.